src/interface/src/app/map/map.component.ts

```diff
@@ -641,6 +649,11 @@ export class MapComponent implements AfterViewInit, OnDestroy, OnInit, DoCheck {
         this.mapManager.addDrawingControl(this.maps[mapIndex].instance!);
         this.mapManager.hideClonedDrawing(this.maps[mapIndex]);
       }
+      setTimeout(() => {
```

Comment:
this ain't pretty but its what we where already doing on `changeMapCount`. Seems like there is a larger issue in the interaction between leaflet and angular that we are patching around with this